### DISCUSSION TITLE
Rote Status Anzeige Bedeutung

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#957] [General]
Hi,

in den letzten Tagen ist mir jetzt öfter aufgefallen das in der Live Ansicht Status Anzeige ziemlich oft rot ist.
Der Wechselrichter selbst zeigt anscheinend keine Störung da er immer weiter im selben Rhythmus die grüne LED blinken lässt.
Es geht um einen Hoymiles HM-1500 mit 4x 375Wp Modulen.
In der Ereignisanzeige wird nichts angezeigt. Gestern hatte ich die auf die Version 23.5.23 aktualisiert und da war es noch extremer. 
Was mir auch aufgefallen ist, es ist ein Intervall von 10 Sekunden zur Aktualisierung eingestellt manchmal braucht es aber bis zu 30 Sekunden oder mehr um wieder auf 0 zu springen.
Was könnte das Problem sein mit der roten Anzeige?

**Comment by DPO99** [comment]
Da kann es viele Ursachen geben...sendest du MQTT Befehle an die DTU? Ansonsten können es auch Verbindungsthemen sein...unterm Strich ist die Anzeige aber nicht so dramatisch. 20 sek. sind m.e. okay. Befehle werden ja trotzdem gesendet, auch wenn die Live-View des WR nicht permanent aktuell ist. Und da man ja eh nicht 24/7 permanent auf die Anzeige schaut fallen solche Sachen eigentlich auch nicht negativ auf.

**Comment by jblond4711** [comment]
Hi,

MQTT ist deaktiviert. Der Standort vom Wechselrichter und der DTU sind unverändert. Was sich ändert ist die Software auf der DTU und das Wetter.
In den nächsten Tagen werde ich mal ein nRF24 mit externer Antenne ausprobieren. Würde mich aber wundern wenn es daran liegen sollte da die Entfernung DTU -> HM-1500 unter 3 Meter ist.
Solange der Wechselrichter aber immer munter grün blinkt sollte es ja wirklich kein Problem geben.

**Reply by DPO99** [reply]
Ja Wie gesagt 30 Sekunden würden mich jetzt nicht nervös machen, es geht ja auch nichts verloren. Und wenn der HM mal rot blinkt aufgrund eines Fehlers dann kann die DTU auch nix retten ;). Und selbst wenn die DTU 120 Sekunden brauchen würde, wäre das dem Hm auch egal, da er die DTU nicht zum arbeiten braucht.

**Comment by tbnobody** [comment]
Und du weißt halt auch nicht was sich in deiner Nachbarschaft verändert hat. 
z.B. https://github.com/tbnobody/OpenDTU/discussions/940#discussioncomment-5960403

Die NRF24 Module von eByte (die mit Shield) sind außerdem viel zuverlässiger als die etwas günstigeren.

**Comment by jblond4711** [comment]
Dann werde ich mir mal ein eByte Modul besorgen.

Edit: Das eByte Modul ist jetzt verbaut. Dann werde ich die Sache jetzt mal beobachten und in ein paar Tagen berichten.

**Comment by jblond4711** [comment]
Nach zwei Tagen intensiver Beobachtungen nehme ich mir mal die Freiheit um zu sagen das seit das eByte Modul verbaut ist ich keine rote Anzeige mehr hatte und auch auf dem Display war keine „Offline“ Anzeige mehr zu sehen.
Nochmals danke für den Tipp.
